### PR TITLE
Feature/development tools jest

### PR DESCRIPTION
**Pull request by tomvik**
### What does this PR do?

- Adds Jest support
- Creates a simple test for the App component
- Fixes some import path issues
- Fixes some eslint issues in the previous files

#### Related issues

#6

#### Merge Checklist:

- [x] The branch is up-to-date (i.e. rebased) with the base branch
- [x] No code style issues

### CODE REVIEW

**Comment by tomvik** [comment]
@RicardoChapaRomero This is ready for review. Thanks and have fun!

**Comment by tomvik** [comment]
@RicardoChapaRomero ping

File: src/components/About/AboutHeader/AboutHeader.js [hunk 1]
```diff
@@ -1,56 +1,58 @@
-import React, { Component } from 'react';
+import React, { Component } from 'react'
 import Carousel from 'react-bootstrap/Carousel'
-import HeaderImg1 from 'images/about/about1.png';
-import HeaderImg2 from 'images/about/about2.jpg';
-import './AboutHeader.css';
+import HeaderImg1 from '../../../images/about/about1.png'
```
Comment (RicardoChapaRomero) [inline src/components/About/AboutHeader/AboutHeader.js:3]: Please delete the relative path of images. That is already fixed using the env variables. See README to enable them
```suggestion
import HeaderImg1 from 'images/about/about1.png'
```

File: src/components/About/AboutTimeline/AboutSingleTimelineEvent/AboutSingleTimelineEvent.js [hunk 2]
```diff
@@ -1,26 +1,26 @@
-import React, { Component } from 'react';
-import { VerticalTimelineElement } from 'react-vertical-timeline-component';
-import placeholder from 'images/placeholder-rectangle.png';
-import 'react-vertical-timeline-component/style.min.css';
-import './AboutSingleTimelineEvent.css';
+import React, { Component } from 'react'
+import { VerticalTimelineElement } from 'react-vertical-timeline-component'
+import placeholder from '../../../../images/placeholder-rectangle.png'
```
Comment (RicardoChapaRomero) [inline src/components/About/AboutTimeline/AboutSingleTimelineEvent/AboutSingleTimelineEvent.js:3]: Delete path here too.

File: src/components/Candidates/Candidates.js [hunk 3]
```diff
@@ -1,26 +1,26 @@
-import React, { Component } from 'react';
-import CandidatesHeader from './CandidatesHeader/CandidatesHeader.js'
-import CandidatesCarousel from './CandidatesCarousel/CandidatesCarousel.js'
-import CandidatesInfo from './CandidatesInfo/CandidatesInfo.js'
-import CandidatesJoinUs from './CandidatesJoinUs/CandidatesJoinUs.js'
-import Footer from 'components/Footer/Footer';
-import positionsData from 'data/positions.json';
-import './Candidates.css';
+import React, { Component } from 'react'
+import positionsData from '../../data/positions.json'
```
Comment (RicardoChapaRomero) [inline src/components/Candidates/Candidates.js:2]:
```suggestion
import positionsData from 'data/positions.json'
```

File: src/components/Candidates/CandidatesCarousel/CandidatesCarousel.js [hunk 4]
```diff
@@ -1,44 +1,44 @@
-import React, { Component } from 'react';
+import React, { Component } from 'react'
 import Carousel from 'react-bootstrap/Carousel'
-import CarouselImg1 from 'images/candidates/candidates1.png';
-import CarouselImg2 from 'images/candidates/candidates2.png';
-import CarouselImg3 from 'images/candidates/candidates3.png';
-import './CandidatesCarousel.css';
+import CarouselImg1 from '../../../images/candidates/candidates1.png'
```
Comment (RicardoChapaRomero) [inline src/components/Candidates/CandidatesCarousel/CandidatesCarousel.js:3]:
```suggestion
import CarouselImg1 from 'images/candidates/candidates1.png'
```

File: src/components/Members/MembersGrid/MemberModal/MemberModal.js [hunk 5]
```diff
@@ -1,231 +1,256 @@
...
-import { MEDIUM_WIDTH } from 'constants.js';
-import './MemberModal.css';
+import React, { Component } from 'react'
+import { Row, Col } from 'react-bootstrap'
+import Button from '@material-ui/core/Button'
+import CloseIcon from '@material-ui/icons/Close'
+import Icon from '@material-ui/core/Icon'
+import IconButton from '@material-ui/core/IconButton'
+import placeholder from '../../../../images/placeholder-rectangle.png'
+import { MEDIUM_WIDTH } from '../../../../constants'
```
Comment (RicardoChapaRomero) [inline src/components/Members/MembersGrid/MemberModal/MemberModal.js:8]:
```suggestion
import { MEDIUM_WIDTH } from 'constants'
```

**Review by RicardoChapaRomero — APPROVED** [review]
Excellente Work Tommy! NIT: Just fix the relative paths on all files and it should be good to merge! If you have any doubts about this you can refer to @Josecisneros001

**Pull request merged**